modules/theme/src/themes/wso2is/modules/dropdown.overrides

```diff
@@ -16,7 +16,7 @@
 
     &.disabled {
         opacity: 1;
-
+        background-color: @DisabledDropdownBgColor !important;
```

Comment:
Let's maintain a consistent style for the `disabled` state in all input fields

Reply:
Let's track that with https://github.com/wso2/product-is/issues/17689